demo/src/app/components/+datepicker/datepicker-section.list.ts

```diff
@@ -63,6 +64,16 @@ export const demoComponentContent: ContentSection[] = [
         `,
         outlet: DemoDatePickerCustomFormatComponent
       },
+      {
+        title: 'Manual datepicker',
```

Comment:
This name doesn't reflect what demo is actually doing. Correct name is `Hide on scroll`